django_project/realtime/management/commands/rerunlasthazard.py

```diff
@@ -22,41 +22,38 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
 
-        root_logger = logging.getLogger('')
-        root_logger.setLevel(logging.INFO)
-
         # Regenerate Earthquake
         try:
-            LOGGER.info('Regenerate Last EQ')
```

Comment:
Why this change? Or doing the print in the logging system? Just curious

Reply:
I forgot that management command usually being run from the shell, so people usually want to see the output directly in the shell. Meanwhile, logger can redirect it into anything, for example storing it on files depending on the log handler